addons/website/controllers/form.py

```diff
@@ -183,7 +184,9 @@ def extract_data(self, model, values):
             elif field_name != 'context':
                 custom_fields.append((field_name, field_value))
 
-        data['custom'] = "\n".join([u"%s : %s" % v for v in custom_fields])
+        # Store the decoded version of the field labels and their corresponding
+        # values.
+        data['custom'] = "\n".join([u"{} : {}".format(werkzeug.urls.url_unquote_plus(v[0]), v[1]) for v in custom_fields])
```

Comment:
`encodeURIComponent` on the client side does not use the "plus" encoding, is that really ok decoding with it? Also are you sure you cannot decode on the client side?